### PR TITLE
Relax banning when POA miners mine out of turn / Check slot in PoaMiner

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs [hunk 1]
```diff
@@ -100,15 +96,15 @@ public override async Task RunAsync(RuleContext context)
...
 
                         break;
                     };
                 }
                 catch (Exception) { }
 
-                this.Logger.LogDebug("(-)[INVALID_SIGNATURE]");
-                PoAConsensusErrors.InvalidHeaderSignature.Throw();
+                //this.Logger.LogDebug("(-)[INVALID_SIGNATURE]");
+                //PoAConsensusErrors.InvalidHeaderSignature.Throw();
```
Comment (quantumagi) [inline src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs:107]: hmm.. Did not expect this getting disabled. In the interest of block chain integrity it's probably not a good idea.

**Review by quantumagi — CHANGES_REQUESTED** [review]
See comment.

File: src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs (hunk 1 above)
Comment (fassadlr) [reply src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs:107]: Ah sorry you are right, cause we set the InsuffcientInformation we wont ban.

**Review by quantumagi — APPROVED** [review]

**Comment by fassadlr** [comment]
Closing as this PR is not the right fix,

**Pull request closed without merging**